src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/ActiveDirectoryReplicationMetaData.cs

```diff
@@ -19,14 +19,14 @@ internal ActiveDirectoryReplicationMetadata(DirectoryServer server)
             _nameTable = Hashtable.Synchronized(tempNameTable);
         }
 
-        public AttributeMetadata this[string name]
+        public AttributeMetadata? this[string name]
         {
             get
             {
                 string tempName = name.ToLowerInvariant();
                 if (Contains(tempName))
                 {
-                    return (AttributeMetadata)InnerHashtable[tempName];
+                    return (AttributeMetadata)InnerHashtable[tempName]!;
```

Comment:
this will always be non-null here since we check if hash table contains it first, it doesn't really matter which one in here since it won't affect nullability